_static/js/annotated.js

```diff
@@ -0,0 +1,66 @@
+// $( document ).ready(function() {
+//     codecomment = document.getElementsByClassName("codecomment");
+
```

Comment:
`Annotated.js` should come from the `a-plus-rst-tools` and thus, it shouldn't be manually added in the course.

Reply:
yes, I will remove those files once the Aalto-LeTech/a-plus-rst-tools#82 is merged